pages/index.tsx

```diff
@@ -1,24 +1,30 @@
-import type { NextPage } from 'next';
-import { useEffect, useState } from 'react';
+import fs from 'node:fs';
+import type { NextPage, GetStaticProps } from 'next';
+import type { DualPost } from '../lib/types';
 import Chrome from '../components/Chrome';
 import HeroBlock from '../components/HeroBlock';
 import RecentPostsBlock from '../components/RecentPostsBlock';
 import { getAllPosts } from '../lib/getAllPosts';
-import { DualPost } from '../lib/types';
+import { podcastXml } from '../lib/podcast';
 
-const Home: NextPage = () => {
-  const [allPosts, setAllPosts] = useState<DualPost[]>([]);
+export const getStaticProps: GetStaticProps = async () => {
+  const allPosts = await getAllPosts();
+  const enPosts = allPosts.map((dual) => dual.en);
+  const esPosts = allPosts.map((dual) => dual.es);
+  fs.writeFileSync(`./public/podcast.en.rss`, podcastXml(enPosts));
+  fs.writeFileSync(`./public/podcast.es.rss`, podcastXml(esPosts));
```

Comment:
here's where i write the xml files into the public dir. (read a couple blog posts, that's what they recommend, and it works).